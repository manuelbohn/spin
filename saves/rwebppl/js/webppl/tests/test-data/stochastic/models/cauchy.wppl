var model = function() {
  var x = cauchy(2, 1.5);
  return 1 < x && x <= 3;
};
