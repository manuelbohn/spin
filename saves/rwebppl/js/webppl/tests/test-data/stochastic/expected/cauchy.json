{
  "hist": {
    "true": 0.3743,
    "false": 0.6257
  }
}
